genie-client/src/main/java/com/netflix/genie/client/JobClient.java

```diff
@@ -574,7 +574,7 @@ public JobStatus waitForCompletion(final String jobId, final long blockTimeout,
             if (System.currentTimeMillis() - startTime < blockTimeout) {
                 Thread.sleep(pollTime);
             } else {
-                throw new GenieTimeoutException("Timed out waiting for job to finish");
+                throw new GenieTimeoutException("Timed out waiting for job to finish: " + jobId);
```

Comment:
This happens in the logs but the code is on the client side. Are you sure this is the error log message you think it is?

Reply:
Yes, smoke tests and gradle plugin test for example print this but no idea what the corresponding job is.